edenia/eden-spend-explorer
# feat(webapp): display current distribution rate

### Display the current distribution rate

### What does this PR do?

- Resolve #369 

### Steps to test

1. run the project and  open it in localhost:3000
2. Go to income and see the next distribution amount 

![image](https://user-images.githubusercontent.com/56526123/211661489-51a8f25b-6eb4-4547-a30d-4cafb3aa3fa1.png)
